extensions/amp-story/1.0/_locales/es-419.js

```diff
@@ -73,19 +73,19 @@ export default {
     string: 'Tumblr',
   },
   [LocalizedStringId.AMP_STORY_SHARING_PROVIDER_NAME_TWITTER]: {
-    string: 'Gorjeo',
+    string: 'Twitter',
   },
   [LocalizedStringId.AMP_STORY_SHARING_PROVIDER_NAME_WHATSAPP]: {
-    string: 'Whatsapp',
+    string: 'WhatsApp',
```

Comment:
Good catch. It should be done in other files too, I filed #18286 to track this change